Yafc/Workspace/ProductionTable/ProductionTableView.cs

```diff
@@ -595,12 +595,13 @@ private class ModulesColumn : ProductionTableDataColumn {
         }
 
         private static readonly IComparer<Goods> DefaultVariantOrdering = new DataUtils.FactorioObjectComparer<Goods>((x, y) => (y.ApproximateFlow() / MathF.Abs(y.Cost())).CompareTo(x.ApproximateFlow() / MathF.Abs(x.Cost())));
-        private RecipeRow AddRecipe(ProductionTable table, Recipe recipe) {
+        private RecipeRow AddRecipe(ProductionTable table, Recipe recipe, Goods SelectedFuel = null) {
             RecipeRow recipeRow = new RecipeRow(table, recipe);
             table.RecordUndo().recipes.Add(recipeRow);
-            recipeRow.entity = recipe.crafters.AutoSelect(DataUtils.FavoriteCrafter);
+            EntityCrafter SelectedFuelCrafter = SelectedFuel?.fuelFor.OfType<EntityCrafter>().Where(e => e.recipes.OfType<Recipe>().Any(e => e == recipe)).AutoSelect(DataUtils.FavoriteCrafter);
```

Comment:
Is there a specific reason why this variable starts from a capital letter?

Reply:
Sloppy coding and missing code style i guess. 👀